package.json

```diff
@@ -37,5 +46,9 @@
   "bugs": {
     "url": "https://github.com/sullenor/css-modules-require-hook/issues"
   },
-  "homepage": "https://github.com/sullenor/css-modules-require-hook"
+  "homepage": "https://github.com/sullenor/css-modules-require-hook",
+  "pre-commit": [
+    "lint",
```

Comment:
This line should be removed later when code is fixed, since we already have `pretest`